source/jormungandr/jormungandr/instance.py

```diff
@@ -49,7 +49,7 @@
 from flask import g
 import flask
 import pybreaker
-from jormungandr import georef, planner, schedule, realtime_schedule, ptref
+from jormungandr import georef, planner, schedule, realtime_schedule, ptref, routing
```

Comment:
I don't really like the name ```routing```, but I don't have anything better than streetnetwork